.github/workflows/to.api.contract.tests.yml

```diff
@@ -0,0 +1,116 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Traffic Ops API Contract Tests
+
+env:
+  # alpine:3.13
+  ALPINE_VERSION: sha256:08d6ca16c60fe7490c03d10dc339d9fd8ea67c6466dea8d558526b1330a85930
+
+on:
+  push:
+    paths:
+      - .github/actions/todb-init/**
+      - .github/actions/to-api-contract-tests/**
+      - .github/workflows/to.api.contract.tests.yml
+      - traffic_ops/testing/api_contract/**
+      - traffic_ops/*client/**.go
+      - traffic_ops/testing/api/**.go
+      - traffic_ops/traffic_ops_golang/**.go
```

Comment:
No need to run this GHA if only Go unit tests (e.g., `traffic_ops/testing/api/v5/users_test.go`) were changed. How about:

```yaml
      - lib/go-tc/**.go
      - traffic_ops/**.go
      - '!**_test.go'
```

Reply:
Sure, Will change it.